e2e/staticpvc.go

```diff
@@ -77,7 +77,7 @@ func getStaticPVC(name, pvName, size, ns, sc string, blockPVC bool) *v1.Persiste
 	return pvc
 }
 
-func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock bool) error {
+func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkImgFeat bool) error {
```

Comment:
This is getting ugly... more booleans as argument causes confusing what boolean is 1st/2nd.

Maybe we should start to introduce something like `checkFlags`:

```golang
const (
    defaultChecks = 0x0
    isBlock       = 0x1
    checkImgFeat  = 0x2
)

validateRBDStaticPV(f, rawAppPath, defaultChecks)
validateRBDStaticPV(f, rawAppPath, isBlock|checkImgFeat)
```

This probably makes the code easier to understand? Not needed to change now, but we have to start thinking about cleaning this up.

Reply:
true, i'll send a follow up pr to refractor/clean it up.